packages/editor/src/components/document-outline/index.js

```diff
@@ -149,11 +142,5 @@ export default compose(
 			blocks: getBlocks(),
 			isTitleSupported: get( postType, [ 'supports', 'title' ], false ),
 		};
-	} ),
-	withDispatch( ( dispatch ) => {
-		const { selectBlock } = dispatch( 'core/editor' );
-		return {
-			onSelect: selectBlock,
```

Comment:
I can understand the change to use links for semantics (and for scroll-to behavior?), but is there a particular reason we don't also still want the block to become selected when clicking the item?

Reply:
for normal links they do become selected